outer-scripts/kubectl-kubestellar-prep_for_syncer

```diff
@@ -131,7 +134,13 @@ else
     kubectl ws "${kubectl_flags[@]}" "$imw"
 fi
 
-mbwsname=$(kubectl "${kubectl_flags[@]}" get synctargets.edge.kubestellar.io "$stname" -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}-mb-{.metadata.uid}")
+if [ "$silent" == "true" ]; then
+    kubectl ws "${kubectl_flags[@]}" "$espw" > /dev/null
+else
+    kubectl ws "${kubectl_flags[@]}" "$espw"
+fi
+
+mbwsname=$(kubectl get "$(kubectl get "${kubectl_flags[@]}" synctargets.edge -oname | grep $stname)" -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}-mb-{.metadata.uid}")
```

Comment:
This is dangerous, it assumes that no SyncTarget's name is a substring of another's.
Why not continue looking in the IMW (IS), where the exact name is known?

Reply:
This is fixed in #1334, using the relation stored by configmaps.

Reply:
done